src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -58,6 +73,12 @@ public void onInteract(PlayerRightClickEvent e) {
                     return;
                 }
 
+                // Check if the recipe of the item is disabled.
+                if (e.getPlayer().getWorld().getGameRuleValue(GameRule.DO_LIMITED_CRAFTING) && hasUnlockedRecipe(e.getPlayer(), e.getItem())) {
+                    SlimefunPlugin.getLocalization().sendMessage(e.getPlayer(), "messages.auto-crafting.recipe-unavailable");
+                    return;
+                }
+
                 // Prevent blocks from being placed, food from being eaten, etc...
                 e.cancel();
```

Comment:
I think these two should be swapped. (`e.cancel()`) should come before the if-block here

Reply:
Okay.